app/src/main/java/com/x8bit/bitwarden/data/autofill/util/ViewNodeExtensions.kt

```diff
@@ -6,9 +6,12 @@ import com.x8bit.bitwarden.data.autofill.model.AutofillView
 import com.x8bit.bitwarden.ui.platform.base.util.orNullIfBlank
 
 /**
- * The class name of the android edit text field.
+ * A lost of possible class names of the android edit text field.
  */
-private const val ANDROID_EDIT_TEXT_CLASS_NAME: String = "android.widget.EditText"
+private val ANDROID_INPUT_FIELD_CLASS_NAMES: List<String> = listOf(
+    "android.widget.EditText",
+    "android.widget.AutoCompleteTextView",
```

Comment:
Question about this: `AutoCompleteTextView` subclasses `EditText` so I'm curious if there is a way we can just check that `EditText` is one of the superclasses of the current class. This is likely an issue with `AppCompatEditText` as well.

Reply:
hmm would something like this work? 
```kotlin
private val AssistStructure.ViewNode.isInputField: Boolean
    get() {
        val kClass: KClass<*> = className?.let { Class.forName(it).kotlin }
        return (kClass as? EditText) != null
    }
```
testing may be hell though

Reply:
Oh that's a pretty interesting idea if that works. Worth a shot! Otherwise I looked into this a little more and this `className` is getting derived from the `getAccessibilityClassName` of the class and so far the only subclasses of `EditText` that override that value that I can find are:

- `AutoCompleteTextView`
- `MultiAutoCompleteTextView `

But your approach would cover all the bases and be future-proof so I'd go in that direction if possible.

Reply:
I did some testing and profiled it to make sure we weren't incurring to much reflection overhead and seems fine.

Ended up with this:

```kotlin
private val AssistStructure.ViewNode.isInputField: Boolean
    get() {
        val isEditText = className
            ?.let { Class.forName(it) }
            ?.let { EditText::class.java.isAssignableFrom(it) } == true
        return isEditText || htmlInfo.isInputField
    }
```